apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -1564,10 +1566,48 @@ public static Bundle createBundle(
    *     Patient}s, which may contain multiple matching resources, or may also be empty.
    */
   public static Bundle addResourcesToBundle(Bundle bundle, List<IBaseResource> resources) {
+    Set<String> beneIds = new HashSet<String>();
     for (IBaseResource res : resources) {
       BundleEntryComponent entry = bundle.addEntry();
       entry.setResource((Resource) res);
+
+      if (entry.getResource().getResourceType().toString() == "ExplanationOfBenefit") {
```

Comment:
can we use an enum comparison here instead of the string comparison?

Reply:
the change was not exactly what I had in mind. Any reason not to do this:

      if (entry.getResource().getResourceType() == ResourceType.ExplanationOfBenefit) {

?